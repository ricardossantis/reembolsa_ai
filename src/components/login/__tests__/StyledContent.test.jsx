import React from "react";
import renderer from "react-test-renderer";
import StyledContent from "../styled-content";

describe("Render Test", () => {
  it("renders StyledContent", () => {
    const component = renderer.create(<StyledContent />).toJSON();
    expect(component).toMatchSnapshot();
  });
});
