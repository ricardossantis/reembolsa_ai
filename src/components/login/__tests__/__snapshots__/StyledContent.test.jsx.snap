// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Test renders StyledContent 1`] = `
<div
  className="sc-bdnylx jRNaPf"
/>
`;
